File: FirstApp/Extras/rough1.vb
Imports System.Data.OleDb
Imports System.IO

Public Class rough1
    Private Sub rough1_Load(sender As Object, e As EventArgs) Handles MyBase.Load

        Dim schemaTable As DataTable = myconnection.GetOleDbSchemaTable(OleDbSchemaGuid.Columns,
                                                                        New Object() {Nothing, Nothing, "Stock_Data", Nothing})

        For Each row As DataRow In schemaTable.Rows
            MessageBox.Show(row("COLUMN_NAME"))
        Next

    End Sub
End Class

File: FirstApp/Extras/rough1.cs
using System;
using System.Data;
using System.Data.OleDb;
using System.Windows.Forms;
using Microsoft.VisualBasic.CompilerServices;

namespace FirstApp
{

    public partial class rough1
    {
        public rough1()
        {
            InitializeComponent();
        }
        private void rough1_Load(object sender, EventArgs e)
        {

            var schemaTable = ProjectModule.myconnection.GetOleDbSchemaTable(OleDbSchemaGuid.Columns, new object[] { null, null, "Stock_Data", null });

            foreach (DataRow row in schemaTable.Rows)
                MessageBox.Show(Conversions.ToString(row["COLUMN_NAME"]));

        }
    }
}